Mage.Sets/src/mage/cards/s/StrefanMaurerProgenitor.java

```diff
@@ -0,0 +1,215 @@
+                false,
+                "When Strefan attacks, "
+        ));
+    }
+
+    private StrefanMaurerProgenitor(final StrefanMaurerProgenitor card) { super(card);}
+
+    @Override
+    public Card copy() {
+        return new StrefanMaurerProgenitor(this);
+    }
+}
+
+class PlayVampireEffect extends OneShotEffect {
+
+    private static final FilterCreatureCard vampireCardFilter = new FilterCreatureCard();
+    static { vampireCardFilter.add(SubType.VAMPIRE.getPredicate()); }
+
+    public PlayVampireEffect() {
+        super(Outcome.PutCreatureInPlay);
+        staticText = "you may put a Vampire card from your hand onto the battlefield tapped and attacking. " +
+                "It gains indestructible until end of turn.";
+    }
+
+    private PlayVampireEffect(final PlayVampireEffect effect) { super(effect); }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player player = game.getPlayer(source.getControllerId());
+        if (player == null) { return false; }
+
+        TargetCard target = new TargetCardInHand(0, 1, vampireCardFilter);
+        if (!player.choose(outcome, player.getHand(), target, game)) { return false; }
+
+        Card card = game.getCard(target.getFirstTarget());
+        if (card == null) { return false; }
+
+        player.moveCards(card, Zone.BATTLEFIELD, source, game, true, false, true, null);
+
+        Permanent permanent = game.getPermanent(card.getId());
+        if (permanent == null) { return false; }
+
+        game.getCombat().addAttackingCreature(permanent.getId(), game);
+
+        // Gains indestructable until end of turn
+        ContinuousEffect effect = new GainAbilityTargetEffect(IndestructibleAbility.getInstance(), Duration.EndOfTurn);
+        effect.setTargetPointer(new FixedTarget(permanent.getId(), game));
+        game.addEffect(effect, source);
+
+        return true;
+    }
+
+    @Override
+    public PlayVampireEffect copy() { return new PlayVampireEffect(this); }
+}
+
+class CreateBloodTokenEffect extends OneShotEffect {
+
+    public CreateBloodTokenEffect() {
+        super(Outcome.Benefit);
+        staticText = "create a Blood token for each player who lost life this turn";
+    }
+
+    private CreateBloodTokenEffect(final CreateBloodTokenEffect effect) { super(effect); }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        if (controller == null) { return false; }
+
+        int playerCount = NumberPlayersLostLifeDynamicValue.instance.calculate(game, source, this);
+
+        if (playerCount > 0) {
+            new BloodToken().putOntoBattlefield(playerCount, game, source, controller.getId());
+        }
+        return playerCount > 0;
+    }
+
+    @Override
+    public CreateBloodTokenEffect copy() { return new CreateBloodTokenEffect(this); }
+}
+
+// From Belbe, Corrupted Observer
+class PlayersWhoLifeLostWatcher extends Watcher {
```

Comment:
Can you not use the `PlayerLostLifeWatcher` (part of common watchers) for this?

Reply:
I looked into it and didn't see it returning the number of players who lost life. But I guess I could iterate through all players, call `getLifeLost`, and then count how many return something greater than 0.

Reply:
Yea, I think that's the way I would do it.  Just reduces the number of watcher.  `PlayerLostLifeWatcher` is automatically run for every game so I think it's best to use that if you can.

Reply:
Unrelated, but should probably double back and see if Belbe, Corrupted Observer can also have its custom watcher removed.